tests/core/test_marc2.py

```diff
@@ -0,0 +1,15 @@
+from core.mirror import MirrorUploader
+
+
+class TestMiscellaneous:
+    def test_mirror_uploader_implementations_are_being_loaded(self):
+        """
+        This test verifies that the two S3 mirror implementations are being loaded when the MARCExporter is
+        imported.  It was not added to tests/core/test_marc.py because that test causes the implementations to be
+        loaded since it references the core.s3 package directly.
+        :return:
```

Comment:
The `:return:` here seems extraneous.

Also, not a big deal, but to make your comments a little more readable, it would be helpful to wrap them at no more than 80 columns, where possible. I usually aim for around 72. Is that because I'm an old FORTRAN guy? Maybe?

Reply:
No problem.